lib/src/main/java/co/nimblehq/alpine/lib/nfc/NfcManager.kt

```diff
@@ -0,0 +1,226 @@
+@file:JvmName("NfcManager")
+
+package co.nimblehq.alpine.lib.nfc
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.nfc.Tag
+import android.nfc.tech.IsoDep
+import androidx.annotation.WorkerThread
+import co.nimblehq.alpine.lib.mrz.MrzInfo
+import co.nimblehq.alpine.lib.model.nfc.*
+import net.sf.scuba.smartcards.CardService
+import org.jmrtd.BACKey
+import org.jmrtd.PassportService
+import org.jmrtd.lds.CardSecurityFile
+import org.jmrtd.lds.PACEInfo
+import org.jmrtd.lds.icao.*
+import org.jmrtd.lds.iso19794.FaceImageInfo
+import org.jmrtd.lds.iso19794.FingerImageInfo
+
+interface NfcReader {
+    @WorkerThread
+    fun readNfc(tag: Tag, mrzInfo: MrzInfo, timeout: Int = ISO_DEP_TIMEOUT_IN_MILLIS): PassportInfo?
+
+    companion object {
+        @JvmStatic
+        fun newInstance(context: Context): NfcReader = NfcReaderImpl(context)
+    }
+}
+
+private const val ISO_DEP_TIMEOUT_IN_MILLIS = 60000
+
+private class NfcReaderImpl(private val context: Context) : NfcReader {
```

Comment:
Should it be `NfcManager` and `NfcManagerImpl`
```suggestion
private class NfcManagerImpl(private val context: Context) : NfcManager {
```

Reply:
Or maybe we should rename the file instead since the only method in this class is `readNfc()` 🤔

Reply:
@ryan-conway @Wadeewee explained [here](https://github.com/nimblehq/alpine/pull/8#discussion_r924032166). I think keeping the Kotlin and JVM file names (`@file:JvmName("NfcManager")`) the same is not a bad idea, right? It's not mandatory, just for consistency. Also, unlike Java, Kotlin allows you to keep related classes, interfaces, enums, functions, etc. in files.

Reply:
@AVI5HEK we could always update that line too 😉 
```kotlin
@file:JvmName("NfcReader")
```

Reply:
@ryan-conway that's what I did but the other way - updated `@file:JvmName("NfcReader")` to `@file:JvmName("NfcManager")` and named the Kotlin file the same. 🤣 

I guess moving the interface and the implementation to separate files will make it less confusing.